tests/rec/test_gender.py

```diff
@@ -95,23 +95,29 @@ def setUp(self):
         self.jane_doe = api.add_identity(self.ctx,
                                          name='Jane Doe',
                                          source='scm')
-        self.smith = api.add_identity(self.ctx,
-                                      name='Smith',
-                                      source='scm')
+        self.john = api.add_identity(self.ctx,
+                                     name='John',
+                                     source='scm')
         self.no_name = api.add_identity(self.ctx,
                                         email='email@example.com',
                                         source='scm')
+        self.double_space = api.add_identity(self.ctx,
+                                            name='John  Smith',
+                                            source='scm')
+        self.initial = api.add_identity(self.ctx,
+                                        name='J Smith',
+                                        source='scm')
 
     @httpretty.activate
-    def test_recommend_gender(self):
-        """Check if it returns a gender for valid names"""
+    def test_recommend_gender_strict(self):
+        """Check if it returns a gender for strict valid names"""
 
         setup_genderize_server()
 
-        uuids = [self.john_smith.uuid, self.jane_doe.uuid]
+        uuids = [self.john_smith.uuid, self.jane_doe.uuid, self.double_space.uuid]
```

Comment:
Can we add here the `self.initial` individual and then we can test nothing is returned for that individual? We can also added to line 167.

Reply:
I'm not sure it's needed, `test_invalid_name_not_strict` and `test_invalid_name_strict` already check that nothing is returned for `self.initial`.

Reply:
I know. It's just to check that there's no a side effect when an invalid name is passed with valid names. In those tests we only pass one value.